app/src/main/java/com/simplecity/amp_library/ui/drawer/DrawerPresenter.java

```diff
@@ -111,15 +113,15 @@ public void onOverflowClick(View v, Playlist playlist) {
                                 })
                                 .toList())
                         .observeOn(AndroidSchedulers.mainThread())
+                        .takeUntil(lifecycleProvider.onDestroyed())
```

Comment:
This `Observable` is added to a list of `Disposables` via `addDisposable()`, which are disposed of in `unbindView()`. I don't see the benefit of the lifecycle callbacks here - `takeUntil(onDestroyed())` is redundant if we're disposing in `unbindView`.